### PR TITLE
Fix: Conflict when multiple attributes in objectInfo event details ha…

### PR DESCRIPTION
**Pull request by klinakerdisy**
…ve the same printNames.

### CODE REVIEW

File: src/cadenza.js [hunk 1]
```diff
@@ -1104,7 +1104,7 @@ function array(/** @type unknown */ value) {
 /** @typedef {CadenzaEvent<'editGeometry:ok', {geometry: Geometry}>} CadenzaEditGeometryOkEvent - When the user submitted the geometry. */
 /** @typedef {CadenzaEvent<'editGeometry:cancel'>} CadenzaEditGeometryCancelEvent - When the user cancelled the geometry editing. */
 /** @typedef {CadenzaEvent<'error', {type: string, message?: string}>} CadenzaErrorEvent - An error event that is mapped to a {@link CadenzaError} */
-/** @typedef {CadenzaEvent<'objectInfo', {layer: WorkbookLayerPath, objectInfos: {selectionIndex: number, formattedValues: Record<string, string>}[]}>} CadenzaObjectInfoEvent - When the user opened the object info flyout. */
+/** @typedef {CadenzaEvent<'objectInfo', {layer: WorkbookLayerPath, objectInfos: {selectionIndex: number, elements: {attributePrintName: Record<string, string>, formattedValues: Record<string, string>}[]}}>} CadenzaObjectInfoEvent - When the user opened the object info flyout. */
```
Comment (dalvarezmartinez1) [inline src/cadenza.js:1107]: how is only a type improvement the fix for this bug? Shouldn't there be some change in code as well?

Comment (jkissel) [reply src/cadenza.js:1107]: The event is sent from Cadenza, i.e. the related code change is there.

Comment (klinakerdisy) [reply src/cadenza.js]: Like jkissel said.

The type was indeed wrong..

Comment (jkissel) [reply src/cadenza.js]: It's still wrong, because it's only _one_ `formattedValue`.

**Review by jkissel — CHANGES_REQUESTED** [review]
objectInfo payload type is still wrong.

**Review by jkissel — APPROVED** [review]

**Pull request merged**